charts/jupyter-with-spark/README.md

```diff
@@ -24,33 +24,87 @@ This chart is work-in-progress
   The above command will deploy the helm chart and will display instructions to access Jupyter service and Spark UI.
   Note that the Spark UI will be available only after you start a SparkContext in Jupyter notebook.
 
-3. Jupyter server URL can be found by running following commands:
+3. Jupyter notebook server URL can be found by running the commands displayed after running `helm install ...` above. 
 
 ```bash
-  $ export JUPYTER_SERVICE_IP=$(kubectl get svc --namespace default jupyter-notebook-jupyter-with-spark -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-  $ echo http://$JUPYTER_SERVICE_IP:{{ .Values.jupyterService.port }}
+  $ export JUPYTER_SERVICE_IP=$(kubectl get svc --namespace default jupyter-jupyter-notebook-ui -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
When including this in the top level readme .... we are simply asking the user to wait for 'kubectl get services' to provide the external IP.